CRITICAL - 2025-01-23 10:09:46 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-23 10:09:46 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-23 10:09:59 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-23 10:10:43 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-23 10:10:44 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-23 10:10:44 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-23 10:10:44 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-23 10:10:44 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-23 10:10:45 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-23 10:12:03 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-23 10:13:23 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(60): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-23 10:13:24 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(60): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-23 10:13:36 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(60): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-23 10:14:02 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(60): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-23 10:16:24 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(60): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-23 04:04:51 --> syntax error, unexpected variable "$data"
in APPPATH\Controllers\Home.php on line 18.
 1 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('home/myview')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('home/myview')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('home/myview')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:05:10 --> syntax error, unexpected variable "$data"
in APPPATH\Controllers\Home.php on line 18.
 1 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('home/myview')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('home/myview')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('home/myview')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:05:49 --> syntax error, unexpected variable "$data1"
in APPPATH\Controllers\Home.php on line 18.
 1 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('home/myview')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('home/myview')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('home/myview')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:05:58 --> syntax error, unexpected variable "$data1"
in APPPATH\Controllers\Home.php on line 18.
 1 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('home/myview')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('home/myview')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('home/myview')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:06:14 --> syntax error, unexpected variable "$data"
in APPPATH\Controllers\Home.php on line 19.
 1 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('home/myview')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('home/myview')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('home/myview')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:06:56 --> syntax error, unexpected variable "$data"
in APPPATH\Controllers\Home.php on line 18.
 1 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('home/myview')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('home/myview')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('home/myview')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:07:58 --> syntax error, unexpected variable "$data"
in APPPATH\Controllers\Home.php on line 18.
 1 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('home/myview')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('home/myview')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('home/myview')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:07:59 --> syntax error, unexpected variable "$data"
in APPPATH\Controllers\Home.php on line 18.
 1 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('home/myview')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('home/myview')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('home/myview')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:29:38 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:30:11 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:31:25 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:32:48 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:32:50 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:32:50 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:32:51 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:32:51 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:32:59 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:33:00 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:33:00 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:33:00 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:33:00 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:33:00 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:33:00 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:33:01 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:34:39 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:34:40 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:34:41 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:34:47 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:38:06 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:38:07 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:38:07 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:38:08 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:38:08 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:38:08 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:38:27 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:38:28 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:38:48 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:40:53 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\Service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:41:49 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\Service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:41:50 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\Service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:41:51 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\Service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:41:51 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\Service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:41:51 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\Service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:41:51 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\Service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:41:52 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\Service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:43:41 --> full_ss5_v2 is not a valid database connection group.
in SYSTEMPATH\Database\Config.php on line 70.
 1 APPPATH\Models\Mydev_model.php(19): CodeIgniter\Database\Config::connect('full_ss5_v2')
 2 APPPATH\Controllers\Service.php(13): App\Models\Mydev_model->__construct()
 3 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\Service->__construct()
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:50:11 --> syntax error, unexpected identifier "parent"
in APPPATH\Controllers\BaseController.php on line 53.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Controllers\\BaseController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\BaseController')
 3 APPPATH\Controllers\Service.php(8): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\BaseController')
 4 SYSTEMPATH\Router\AutoRouter.php(147): include_once('APPPATH\\Controllers\\Service.php')
 5 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('service')
 6 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('service')
 7 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('service')
 8 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 04:50:13 --> syntax error, unexpected identifier "parent"
in APPPATH\Controllers\BaseController.php on line 53.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Controllers\\BaseController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\BaseController')
 3 APPPATH\Controllers\Service.php(8): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\BaseController')
 4 SYSTEMPATH\Router\AutoRouter.php(147): include_once('APPPATH\\Controllers\\Service.php')
 5 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('service')
 6 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('service')
 7 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('service')
 8 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:15:19 --> Object of class stdClass could not be converted to string
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 06:17:42 --> syntax error, unexpected token "}", expecting "," or ";"
in APPPATH\Controllers\Service.php on line 35.
 1 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('service')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('service')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('service')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:17:50 --> Object of class stdClass could not be converted to string
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:18:01 --> Array to string conversion
in APPPATH\Controllers\Service.php on line 34.
 1 APPPATH\Controllers\Service.php(34): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'APPPATH\\Controllers\\Service.php', 34)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:19:50 --> Undefined constant "App\Controllers\username"
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:19:51 --> Undefined constant "App\Controllers\username"
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:20:00 --> Object of class stdClass could not be converted to string
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:20:01 --> Object of class stdClass could not be converted to string
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:20:06 --> Array to string conversion
in APPPATH\Controllers\Service.php on line 34.
 1 APPPATH\Controllers\Service.php(34): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'APPPATH\\Controllers\\Service.php', 34)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 06:21:08 --> syntax error, unexpected token "=>", expecting ")"
in APPPATH\Controllers\Service.php on line 35.
 1 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('service')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('service')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('service')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:21:15 --> Attempt to read property "username" on array
in APPPATH\Controllers\Service.php on line 35.
 1 APPPATH\Controllers\Service.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "username" on array', 'APPPATH\\Controllers\\Service.php', 35)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:21:27 --> Attempt to read property "username" on array
in APPPATH\Controllers\Service.php on line 35.
 1 APPPATH\Controllers\Service.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "username" on array', 'APPPATH\\Controllers\\Service.php', 35)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:21:28 --> Attempt to read property "username" on array
in APPPATH\Controllers\Service.php on line 35.
 1 APPPATH\Controllers\Service.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "username" on array', 'APPPATH\\Controllers\\Service.php', 35)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:21:28 --> Attempt to read property "username" on array
in APPPATH\Controllers\Service.php on line 35.
 1 APPPATH\Controllers\Service.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "username" on array', 'APPPATH\\Controllers\\Service.php', 35)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:21:28 --> Attempt to read property "username" on array
in APPPATH\Controllers\Service.php on line 35.
 1 APPPATH\Controllers\Service.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "username" on array', 'APPPATH\\Controllers\\Service.php', 35)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:21:28 --> Attempt to read property "username" on array
in APPPATH\Controllers\Service.php on line 35.
 1 APPPATH\Controllers\Service.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "username" on array', 'APPPATH\\Controllers\\Service.php', 35)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:21:28 --> Attempt to read property "username" on array
in APPPATH\Controllers\Service.php on line 35.
 1 APPPATH\Controllers\Service.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "username" on array', 'APPPATH\\Controllers\\Service.php', 35)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 13:25:10 --> Undefined property: stdClass::$citizinID
in APPPATH\Controllers\Service.php on line 36.
 1 APPPATH\Controllers\Service.php(36): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$citizinID', 'APPPATH\\Controllers\\Service.php', 36)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:36:53 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1 in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO 'us...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO 'us...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO 'us...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO 'us...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO 'us...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:36:53 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:37:20 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1 in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO 'us...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO 'us...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO 'us...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO 'us...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO 'us...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:37:20 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:37:25 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1 in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO 'us...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO 'us...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO 'us...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO 'us...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO 'us...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:37:25 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:37:27 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1 in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO 'us...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO 'us...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO 'us...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO 'us...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO 'us...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:37:27 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:37:54 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1 in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO 'us...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO 'us...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO 'us...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO 'us...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO 'us...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:37:54 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:37:54 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1 in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO 'us...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO 'us...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO 'us...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO 'us...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO 'us...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:37:54 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:37:55 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1 in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO 'us...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO 'us...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO 'us...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO 'us...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO 'us...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:37:55 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:37:56 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1 in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO 'us...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO 'us...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO 'us...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO 'us...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO 'us...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:37:56 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:37:56 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1 in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO 'us...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO 'us...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO 'us...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO 'us...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO 'us...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:37:56 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:38:03 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1 in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO 'us...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO 'us...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO 'us...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO 'us...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO 'us...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:38:03 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:39:48 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1 in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO 'us...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO 'us...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO 'us...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO 'us...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO 'us...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:39:48 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:40:10 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1 in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO 'us...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO 'us...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO 'us...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO 'us...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO 'us...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:40:10 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''user_info' ('username','password','citizin_id','email','name','surname','add...' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO \'user_info\' (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         values
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:42:54 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''username','password','citizin_id','email','name','surname','address','phone_...' at line 1 in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:42:54 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''username','password','citizin_id','email','name','surname','address','phone_...' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         VALUES
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO user_info (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         VALUES
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:43:32 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''username','password','citizin_id','email','name','surname','address','phone_...' at line 1 in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:43:32 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ''username','password','citizin_id','email','name','surname','address','phone_...' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         VALUES
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(53): App\Models\Mydev_model->execute('INSERT INTO user_info (\'username\',\'password\',\'citizin_id\',\'email\',\'name\',\'surname\',\'address\',\'phone_number\')
         VALUES
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:51:47 --> mysqli_sql_exception: Duplicate entry 'testname123' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:51:47 --> Duplicate entry 'testname123' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname123\',\'Password01!\',\'7894561231233\',\'kun@hot.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:55:01 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:55:01 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:55:30 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:55:30 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:55:32 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:55:32 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:55:57 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:55:57 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:56:03 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:56:03 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:56:32 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:56:32 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:56:32 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:56:32 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:58:14 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:58:14 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:58:15 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:58:15 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:58:16 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:58:16 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:58:16 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:58:16 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:58:16 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:58:16 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:58:16 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:58:16 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:59:45 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:59:45 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 13:59:46 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 13:59:46 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 14:01:24 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 14:01:24 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 14:01:50 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 14:01:50 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 14:02:35 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 14:02:35 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 14:03:25 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 14:03:25 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 14:03:26 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 14:03:26 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 14:03:46 --> mysqli_sql_exception: Duplicate entry 'testname1234' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->index()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 14:03:46 --> Duplicate entry 'testname1234' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 APPPATH\Controllers\Service.php(58): App\Models\Mydev_model->execute('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname1234\',\'a332233d395c415a99c6b16a2348c129\',\'7894561231333\',\'kun@hotm.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 07:14:25 --> syntax error, unexpected token ";"
in APPPATH\Controllers\Service.php on line 72.
 1 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('service')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('service')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('service')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 07:14:35 --> syntax error, unexpected token ";"
in APPPATH\Controllers\Service.php on line 72.
 1 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('service')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('service')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('service')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 14:18:11 --> Attempt to read property "username" on array
in APPPATH\Controllers\Service.php on line 56.
 1 APPPATH\Controllers\Service.php(56): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "username" on array', 'APPPATH\\Controllers\\Service.php', 56)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 14:18:45 --> Attempt to read property "username" on array
in APPPATH\Controllers\Service.php on line 56.
 1 APPPATH\Controllers\Service.php(56): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "username" on array', 'APPPATH\\Controllers\\Service.php', 56)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 14:18:46 --> Attempt to read property "username" on array
in APPPATH\Controllers\Service.php on line 56.
 1 APPPATH\Controllers\Service.php(56): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "username" on array', 'APPPATH\\Controllers\\Service.php', 56)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 14:18:47 --> Attempt to read property "username" on array
in APPPATH\Controllers\Service.php on line 56.
 1 APPPATH\Controllers\Service.php(56): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "username" on array', 'APPPATH\\Controllers\\Service.php', 56)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 14:18:54 --> Attempt to read property "username" on array
in APPPATH\Controllers\Service.php on line 56.
 1 APPPATH\Controllers\Service.php(56): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "username" on array', 'APPPATH\\Controllers\\Service.php', 56)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 14:18:55 --> Attempt to read property "username" on array
in APPPATH\Controllers\Service.php on line 56.
 1 APPPATH\Controllers\Service.php(56): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "username" on array', 'APPPATH\\Controllers\\Service.php', 56)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 14:19:03 --> Array to string conversion
in APPPATH\Controllers\Service.php on line 56.
 1 APPPATH\Controllers\Service.php(56): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'APPPATH\\Controllers\\Service.php', 56)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 07:19:52 --> syntax error, unexpected identifier "var_dump"
in APPPATH\Controllers\Service.php on line 57.
 1 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('service')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('service')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('service')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 14:20:00 --> Attempt to read property "username" on array
in APPPATH\Controllers\Service.php on line 56.
 1 APPPATH\Controllers\Service.php(56): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "username" on array', 'APPPATH\\Controllers\\Service.php', 56)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 14:20:18 --> Attempt to read property "username" on array
in APPPATH\Controllers\Service.php on line 56.
 1 APPPATH\Controllers\Service.php(56): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "username" on array', 'APPPATH\\Controllers\\Service.php', 56)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 14:27:44 --> Array to string conversion
in APPPATH\Controllers\Service.php on line 56.
 1 APPPATH\Controllers\Service.php(56): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'APPPATH\\Controllers\\Service.php', 56)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 14:38:42 --> Undefined array key 0
in APPPATH\Controllers\Service.php on line 59.
 1 APPPATH\Controllers\Service.php(59): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'APPPATH\\Controllers\\Service.php', 59)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 14:38:43 --> Undefined array key 0
in APPPATH\Controllers\Service.php on line 59.
 1 APPPATH\Controllers\Service.php(59): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'APPPATH\\Controllers\\Service.php', 59)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 14:39:29 --> Undefined array key 0
in APPPATH\Controllers\Service.php on line 58.
 1 APPPATH\Controllers\Service.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'APPPATH\\Controllers\\Service.php', 58)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 14:39:30 --> Undefined array key 0
in APPPATH\Controllers\Service.php on line 58.
 1 APPPATH\Controllers\Service.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'APPPATH\\Controllers\\Service.php', 58)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 07:40:03 --> syntax error, unexpected token "}", expecting "," or ";"
in APPPATH\Controllers\Service.php on line 68.
 1 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('service')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('service')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('service')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 14:52:02 --> Call to a member function getResult() on bool
in APPPATH\Models\Mydev_model.php on line 35.
 1 APPPATH\Controllers\Service.php(97): App\Models\Mydev_model->select('INSERT INTO user_info (username,password,citizin_id,email,name,surname,address,phone_number)
         VALUES
         (\'testname12347\',\'a332233d395c415a99c6b16a2348c129\',\'7894561233333\',\'kun@hotma.lon\',\'Hi\',\'LO\',\'gg\',\'02\')')
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 15:01:34 --> Undefined variable $user
in APPPATH\Controllers\Service.php on line 73.
 1 APPPATH\Controllers\Service.php(73): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $user', 'APPPATH\\Controllers\\Service.php', 73)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 15:01:42 --> Undefined variable $user
in APPPATH\Controllers\Service.php on line 73.
 1 APPPATH\Controllers\Service.php(73): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $user', 'APPPATH\\Controllers\\Service.php', 73)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 15:05:08 --> Undefined variable $user
in APPPATH\Controllers\Service.php on line 78.
 1 APPPATH\Controllers\Service.php(78): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $user', 'APPPATH\\Controllers\\Service.php', 78)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 15:08:29 --> Undefined variable $user
in APPPATH\Controllers\Service.php on line 81.
 1 APPPATH\Controllers\Service.php(81): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $user', 'APPPATH\\Controllers\\Service.php', 81)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 15:10:05 --> Undefined variable $email_er
in APPPATH\Controllers\Service.php on line 89.
 1 APPPATH\Controllers\Service.php(89): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $email_er', 'APPPATH\\Controllers\\Service.php', 89)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 15:11:12 --> Undefined variable $email_er
in APPPATH\Controllers\Service.php on line 89.
 1 APPPATH\Controllers\Service.php(89): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $email_er', 'APPPATH\\Controllers\\Service.php', 89)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 15:11:13 --> Undefined variable $email_er
in APPPATH\Controllers\Service.php on line 89.
 1 APPPATH\Controllers\Service.php(89): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $email_er', 'APPPATH\\Controllers\\Service.php', 89)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 15:12:20 --> Undefined variable $email_er
in APPPATH\Controllers\Service.php on line 90.
 1 APPPATH\Controllers\Service.php(90): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $email_er', 'APPPATH\\Controllers\\Service.php', 90)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 08:15:11 --> syntax error, unexpected token ")"
in APPPATH\Controllers\Service.php on line 92.
 1 SYSTEMPATH\Router\Router.php(506): CodeIgniter\Router\AutoRouter->getRoute('service')
 2 SYSTEMPATH\Router\Router.php(205): CodeIgniter\Router\Router->autoRoute('service')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('service')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 15:44:10 --> Array to string conversion
in APPPATH\Controllers\Service.php on line 61.
 1 APPPATH\Controllers\Service.php(61): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'APPPATH\\Controllers\\Service.php', 61)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:25:57 --> Call to undefined function App\Controllers\write_log()
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:26:10 --> Call to undefined function App\Controllers\write_log()
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:26:13 --> Call to undefined function App\Controllers\write_log()
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:29:14 --> Call to undefined function App\Controllers\write_log()
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:31:13 --> Call to undefined function App\Controllers\write_log()
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:31:24 --> Call to undefined function App\Controllers\write_log()
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:35:00 --> Call to undefined function App\Controllers\write_log()
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:36:36 --> include_once(../day7/include/WebConfig.php): Failed to open stream: No such file or directory
in APPPATH\Controllers\Service.php on line 28.
 1 APPPATH\Controllers\Service.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include_once(../day7/include/WebConfig.php): Failed to open stream: No such file or directory', 'APPPATH\\Controllers\\Service.php', 28)
 2 APPPATH\Controllers\Service.php(28): include_once()
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:37:58 --> Call to undefined function App\Controllers\write_log()
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:39:22 --> Call to undefined function App\Controllers\write_log()
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:40:24 --> Call to undefined function App\Controllers\write_log()
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:40:27 --> Call to undefined function App\Controllers\write_log()
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:40:33 --> Call to undefined function App\Controllers\write_log()
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:43:24 --> Call to undefined function App\Controllers\write_log()
in APPPATH\Controllers\Service.php on line 34.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:45:01 --> Call to undefined function App\Controllers\write_log()
in APPPATH\Controllers\Service.php on line 35.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:45:32 --> Call to undefined function App\Controllers\get_client_ip()
in APPPATH\Controllers\Service.php on line 484.
 1 APPPATH\Controllers\Service.php(309): App\Controllers\Service->write_log('{"ret":"101","msg":"success","data":{"username":"hghghghghg","password":"Password123!","c_password":"Password123!","name":"fd","surname":"kk","email":"h@h.com","cid":"1111111111111","phonenumber":"","address":""},"location":"https:\\/\\/together-gladly-airedale.ngrok-free.app\\/home"}')
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 17:45:59 --> mysqli_sql_exception: Duplicate entry 'hghghghghg' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(266): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->process_register()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 17:45:59 --> Duplicate entry 'hghghghghg' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info
        (`username`, `password`, `citizin_id`, `email`, `name`, `surname`, `address`, `phone_number`) 
        VALUES (\'hghghghghg\', \'2c103f2c4ed1e59c0b4e2e01821770fa\', \'1111111111111\', \'h@h.com\', \'fd\', \'kk\',\' \', \'\');
        ')
 2 APPPATH\Controllers\Service.php(266): App\Models\Mydev_model->execute('INSERT INTO user_info
        (`username`, `password`, `citizin_id`, `email`, `name`, `surname`, `address`, `phone_number`) 
        VALUES (\'hghghghghg\', \'2c103f2c4ed1e59c0b4e2e01821770fa\', \'1111111111111\', \'h@h.com\', \'fd\', \'kk\',\' \', \'\');
        ')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
ERROR - 2025-01-23 17:47:06 --> mysqli_sql_exception: Duplicate entry 'hghghghghg' for key 'username' in C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ci4_ss5\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO use...', 0)
#1 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO use...')
#2 C:\xampp\htdocs\ci4_ss5\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO use...')
#3 C:\xampp\htdocs\ci4_ss5\app\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO use...')
#4 C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php(265): App\Models\Mydev_model->execute('INSERT INTO use...')
#5 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(934): App\Controllers\Service->process_register()
#6 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
#7 C:\xampp\htdocs\ci4_ss5\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4_ss5\public\index.php(72): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-23 17:47:06 --> Duplicate entry 'hghghghghg' for key 'username'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 APPPATH\Models\Mydev_model.php(24): CodeIgniter\Database\BaseConnection->query('INSERT INTO user_info
        (`username`, `password`, `citizin_id`, `email`, `name`, `surname`, `address`, `phone_number`) 
        VALUES (\'hghghghghg\', \'2c103f2c4ed1e59c0b4e2e01821770fa\', \'1111111111111\', \'h@h.com\', \'fd\', \'kk\',\' \', \'\');
        ')
 2 APPPATH\Controllers\Service.php(265): App\Models\Mydev_model->execute('INSERT INTO user_info
        (`username`, `password`, `citizin_id`, `email`, `name`, `surname`, `address`, `phone_number`) 
        VALUES (\'hghghghghg\', \'2c103f2c4ed1e59c0b4e2e01821770fa\', \'1111111111111\', \'h@h.com\', \'fd\', \'kk\',\' \', \'\');
        ')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:48:22 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(640): CodeIgniter\Database\BaseConnection->table(null)
 2 SYSTEMPATH\Model.php(801): CodeIgniter\Model->builder()
 3 APPPATH\Controllers\Service.php(265): CodeIgniter\Model->__get('execute')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-23 17:49:29 --> Too few arguments to function App\Models\Mydev_model::execute(), 0 passed in C:\xampp\htdocs\ci4_ss5\app\Controllers\Service.php on line 265 and exactly 1 expected
in APPPATH\Models\Mydev_model.php on line 22.
 1 APPPATH\Controllers\Service.php(265): App\Models\Mydev_model->execute()
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Service->process_register()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(72): CodeIgniter\CodeIgniter->run()
